src/bundle.ts

```diff
@@ -32,14 +33,23 @@ export class BuildBundler extends Transform {
   bundler: Bundler;
   sharedFile: File;
```

Comment:
As this class is exported, how much of this API can we make private and/or readonly?

Also, would it be a breaking change to rename `analyzer` to `buildAnalyzer`, as this class also does use the Analyzer class